client/src/services/local-storage.js

```diff
@@ -0,0 +1,24 @@
+import config from "../config";
+
+export const loadStateFromLocalStorage = () => {
+        const serializedState = localStorage.getItem(config.stateKeyName);
```

Comment:
I would change `stateKeyName` to `storeLocalStorageKey`

Reply:
store?

Reply:
`store` is the object that holds the state, but i meant `stateLocalStorageKey`

Reply:
yeah that's what I want - `stateLocalStorageKey`